presto-docs/src/main/sphinx/migration/from-hive.rst

```diff
@@ -109,3 +109,14 @@ Presto query::
     SELECT student, score
     FROM tests
     CROSS JOIN UNNEST(scores) AS t (score);
+
+Caution with datediff
+---------------------
+
+The Hive ``datediff`` function is declared as ``datediff(string enddate, string
+startdate)``. The equivalent function in ANSI SQL is ``date_diff`` and it is
+declared as ``date_diff(unit, timestamp1, timestamp2) → bigint``. It calculates
```

Comment:
Rather than repeating the documentation here, let's simply link to the function and note the parameters and functionality is different.

Reply:
We could give an example of how to migrate from Hive to Presto.

Reply:
Done ... I kept the example as simple as possible and not build it up into full HiveQL and SQL queries since that just makes it harder to read.